server/server.js

```diff
@@ -64,11 +64,13 @@ app.get('/api/trips', cors(), async (req, res) => {
 // Make the GET request with the country and year (that it's the redirect from the user)
 app.get("/api/holidays", cors(), async (req, res) => {
     country = req.query.country;
+    month = req.query.month;
     year = req.query.year;
```

Comment:
I believe you can also get these variables through [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment):
```suggestion
    const {country, month, year} = req.query;
```